src/SharedType.elm

```diff
@@ -15,3 +15,8 @@ type Tabs
 type KodiTabs
     = Audio
     | Video
+
+
+type Selected
```

Comment:
I think the name could be better. I also think with the approach outlined, it will need to be page specific. So something like `AlbumSorting`. But in the best case, we can move this out of the SharedTypes into the albums pages.

Reply:
I understand your point and will do that.